src/restconfig/src/main/java/org/geoserver/rest/catalog/WorkspaceController.java

```diff
@@ -88,7 +88,7 @@ public RestWrapper workspacesGet() {
     }
 
     @GetMapping(value = "/{workspaceName}")
-    public RestWrapper<WorkspaceInfo> getWorkspace(@PathVariable String workspaceName) {
+    public RestWrapper<WorkspaceInfo> workspaceG(@PathVariable String workspaceName) {
```

Comment:
Looks like there is a mistake here?

Reply:
Yeah should be workspaceGet to agree with the naming convention; fixing now